pyserini/search/faiss/_searcher.py

```diff
@@ -168,10 +168,11 @@ def _mean_pooling(model_output, attention_mask):
         sum_mask = torch.clamp(input_mask_expanded.sum(1), min=1e-9)
         return sum_embeddings / sum_mask
 
-    def encode(self, query: str):
+    def encode(self, query: str, max_length: int = 64):
```

Comment:
is this change going to affect all our other models?

Reply:
No , it won't